src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -53375,38 +53383,43 @@ static void Main()
     }
 
     CL0<string?> M1(CL0<string> x) { throw new System.Exception(); }
+    CL0<string> M2(CL0<string> x) { throw new System.Exception(); }
```

Comment:
This test changed for the same reason as https://github.com/dotnet/roslyn/pull/46953#discussion_r474304674.